pkg/printers/printers.go

```diff
@@ -25,92 +25,102 @@ import (
-			return nil, false, fmt.Errorf("templatefile format specified but no template file given")
+			return nil, fmt.Errorf("templatefile format specified but no template file given")
 		}
 		data, err := ioutil.ReadFile(formatArgument)
 		if err != nil {
-			return nil, false, fmt.Errorf("error reading template %s, %v\n", formatArgument, err)
+			return nil, fmt.Errorf("error reading template %s, %v\n", formatArgument, err)
 		}
 		templatePrinter, err := NewTemplatePrinter(data)
 		if err != nil {
-			return nil, false, fmt.Errorf("error parsing template %s, %v\n", string(data), err)
+			return nil, fmt.Errorf("error parsing template %s, %v\n", string(data), err)
 		}
 		templatePrinter.AllowMissingKeys(allowMissingTemplateKeys)
 		printer = templatePrinter
+
 	case "jsonpath":
 		if len(formatArgument) == 0 {
-			return nil, false, fmt.Errorf("jsonpath template format specified but no template given")
+			return nil, fmt.Errorf("jsonpath template format specified but no template given")
 		}
 		jsonpathPrinter, err := NewJSONPathPrinter(formatArgument)
 		if err != nil {
-			return nil, false, fmt.Errorf("error parsing jsonpath %s, %v\n", formatArgument, err)
+			return nil, fmt.Errorf("error parsing jsonpath %s, %v\n", formatArgument, err)
 		}
 		jsonpathPrinter.AllowMissingKeys(allowMissingTemplateKeys)
 		printer = jsonpathPrinter
+
 	case "jsonpath-file":
 		if len(formatArgument) == 0 {
-			return nil, false, fmt.Errorf("jsonpath file format specified but no template file file given")
+			return nil, fmt.Errorf("jsonpath file format specified but no template file file given")
 		}
 		data, err := ioutil.ReadFile(formatArgument)
 		if err != nil {
-			return nil, false, fmt.Errorf("error reading template %s, %v\n", formatArgument, err)
+			return nil, fmt.Errorf("error reading template %s, %v\n", formatArgument, err)
 		}
 		jsonpathPrinter, err := NewJSONPathPrinter(string(data))
 		if err != nil {
-			return nil, false, fmt.Errorf("error parsing template %s, %v\n", string(data), err)
+			return nil, fmt.Errorf("error parsing template %s, %v\n", string(data), err)
 		}
 		jsonpathPrinter.AllowMissingKeys(allowMissingTemplateKeys)
 		printer = jsonpathPrinter
+
 	case "custom-columns":
 		var err error
 		if printer, err = NewCustomColumnsPrinterFromSpec(formatArgument, decoders[0], noHeaders); err != nil {
-			return nil, false, err
+			return nil, err
 		}
+
 	case "custom-columns-file":
 		file, err := os.Open(formatArgument)
 		if err != nil {
-			return nil, false, fmt.Errorf("error reading template %s, %v\n", formatArgument, err)
+			return nil, fmt.Errorf("error reading template %s, %v\n", formatArgument, err)
 		}
 		defer file.Close()
 		if printer, err = NewCustomColumnsPrinterFromTemplate(file, decoders[0]); err != nil {
-			return nil, false, err
+			return nil, err
 		}
+
 	case "wide":
 		fallthrough
 	case "":
-		return nil, false, nil
+
+		printer = NewHumanReadablePrinter(encoder, decoders[0], options)
```

Comment:
Do we need to move this line up for `wide` only?
I suspect it changes the behavior when not specify `--output` flag.